### DISCUSSION TITLE
Автоматическое заполнение атрибутов запроса PR

### DISCUSSION THREAD

**Discussion by pivovarov-mai** [calculation-methods/M8O-303B-21#17] [Ideas]
В репозиторий добавлено автоматическое заполнение полей **reviewer**, **assignee** и **labels**.
Вам остается только дозаполнить labels метками `Частично/Полностью` и проставить `Milestone`

**Comment by Krukrukruzhka** [comment]
На данный момент автозаполнение assignee и labels не работает
